pom.xml

```diff
@@ -114,6 +114,110 @@
                 <jdk.build.target>22</jdk.build.target>
             </properties>
           </profile>
+          <!--
+          Profile expected to be active most of the time whenever testing what was produced by
+          this project. This includes the providers built and local test artifacts.
+          -->
+          <profile>
+            <id>defaultestenv</id>
```

Comment:
Can we change that to something more descriptive?

Reply:
I updated this id value to be much more descriptive.